### PR TITLE
Adapting Iguassu to the Arrebol json job model. 

### PR DESCRIPTION
**Pull request by emanueljoivo**

### CODE REVIEW

File: src/main/java/org/fogbowcloud/app/core/IguassuController.java [hunk 1]
```diff
@@ -119,11 +119,7 @@ public JDFJob buildJob(String jdfFilePath, User owner) throws CompilerException
 
         String externalOAuthToken = getAccessTokenByOwnerUsername(userName);
 
-        JDFJob jobBuilt = buildJobFromJDFFile(job, jdfFilePath,jobSpec, userName, externalOAuthToken);
-
-        this.createdJobs.put(jobBuilt.getId(), jobBuilt);
-
-        return job;
+        return buildJobFromJDFFile(job, jdfFilePath,jobSpec, userName, externalOAuthToken);
```
Comment (thiagomanel) [inline src/main/java/org/fogbowcloud/app/core/IguassuController.java]: don't we need to put it on the createdJobs anymore? why?

File: src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolJob.java [hunk 2]
```diff
@@ -13,15 +10,12 @@
...
 
     public ArrebolJob(String label, Collection<ArrebolTask> tasks){
         this.jobState = ArrebolJobState.SUBMITTED;
-        this.tasks = new HashMap<>();
-
-        for(ArrebolTask task: tasks) {
-            this.tasks.put(task.getId(), task);
-        }
+        this.tasks = new LinkedList<>();
+        this.tasks.addAll(tasks);
```
Comment (thiagomanel) [inline src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolJob.java]: has LList a constructor that receives another collection as arg?

File: src/main/java/org/fogbowcloud/app/core/IguassuController.java (hunk 1 above)
Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/core/IguassuController.java]: This data structure is no longer used anywhere. It made sense in the blowout context that saved one thread per job, just like the map of finished tasks.

Comment (thiagomanel) [reply src/main/java/org/fogbowcloud/app/core/IguassuController.java]: so, why don't you remove it? It is still declared

Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/core/IguassuController.java]: Done.

File: src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolJob.java (hunk 2 above)
Comment (emanueljoivo) [reply src/main/java/org/fogbowcloud/app/jes/arrebol/models/ArrebolJob.java]: Done.

**Pull request merged**